### ISSUE TITLE
Downloadable APK or Android Play Store Version

### ISSUE THREAD

**Issue by letsgamedev** [FundamentalFrequency/LMN-3-DAW#48]
Hello, I just soldered the keys and the teensy onto the circuit board and tested the whole thing with some music programs on my Android smartphone. And it works! The LMN-3 is recognized as a normal MIDI device.

It would be an interesting possibility if the DAW program could also be available as an apk, provided that the whole thing can be compiled on Android at all. Or then even uploaded to the play store?

A phone has a display, speakers, and a microphone. So you could save the costs for the Rasberry Pi and the display completely. And that would reduce the project costs by around 80 € to 100 €, because everyone has a mobile phone :D

**Comment by stonepreston** [comment]
Yeah getting things working on android/iOS would definitely be nice. iOS will probably be a bit easier as the JUCE CMake integration works fine for iOS but not android. Definitely possible to get working. Tracktion Engine supports android and iOS as well so I dont think there would be too many issues getting it going.

**Comment by coderoid** [comment]
I am currently working on this, @stonepreston do you want to assign this to me?

**Comment by BlackBoxFox** [comment]
@coderoid Seriously holding out hope. Would LOVE to support this development.
